core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,11 +347,16 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants=kubelet.service docker.service
-        After=kubelet.service
+        Wants=kubelet.service docker.service install-kube-system.service
+        After=kubelet.service install-kube-system.service
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=60
+        StartLimitInterval=320
```

Comment:
@dvdthms Thanks for the suggestion 👍 I'll take it into account.

Anyways, for this concrete case, I'm considering to change cfn-signal and install-kube-system to `Type=oneshot` with retrying via shell scripting so that we can control execution order without spamming journal like now.